cheroot/test/test_ssl.py

```diff
@@ -71,11 +72,12 @@
 )
 
 
-class HelloWorldGateway(Gateway):
+class HelloWorldGateway(Gateway_10):
     """Gateway responding with Hello World to root URI."""
 
     def respond(self):
         """Respond with dummy content via HTTP."""
+        type(self).env = self.get_environ()
```

Comment:
Why are you doing this?

Reply:
it was a shortcut to pass the environment back to the test... i've since changed it to pickle it through the response body